ui/install.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+set -x
+set -e
+
+# Mobile API Server repo location
+MOBILE_API_SERVER_REPO=github.com/feedhenry/mobile-apiserver
```

Comment:
given that the mobile-apiserver code is in the same repo as this script now, i think a relative path woudl be better in cases where someone has cloned to a different dir